osu.Game/Overlays/Dashboard/Friends/DashboardDisplay.cs

```diff
@@ -0,0 +1,49 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Allocation;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Game.Online.API;
+
+namespace osu.Game.Overlays.Dashboard.Friends
+{
+    public abstract class DashboardDisplay : CompositeDrawable
```

Comment:
can spotlights use this class? i would hope so.

also xmldoc all public classes also xmldoc all public classesalso xmldoc all public classesalso xmldoc all public classesalso xmldoc all public classesalso xmldoc all public classesalso xmldoc all public classesalso xmldoc all public classesalso xmldoc all public classesalso xmldoc all public classes

what is this supposed to be? when should it be used? why would i use it?

Reply:
I forgot to remove it, my bad.

Reply:
Oh, wait, I did